include/kodlab_mjbots_sdk/robot_base.h

```diff
@@ -82,7 +82,7 @@ namespace kodlab
          * @warning All derivative classes overriding this method should include
          *          a cycle count increment (i.e., `cycle_count_++;`).
          */
-        virtual void Update(){cycle_count_++;}; //TODO remove cycle_count and use time or more intelligently set cycle_count or properly handle softstart
+        virtual void Update(){}; //TODO remove cycle_count and use time or more intelligently set cycle_count or properly handle softstart
```

Comment:
```suggestion
        virtual void Update(){}; 
```

Reply:
fixed in 21511f7a0d652fe35a9fc578d4b151ac0b83c9d4